docs/release-notes/release-notes-0.8.3.md

```diff
@@ -0,0 +1,253 @@
+Joinmarket-clientserver 0.8.3:
+=================
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/releases/tag/v0.8.3>
+
+Please report bugs using the issue tracker at github:
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/issues>
+
+Upgrading
+=========
+
+To upgrade:
+
+(If you are upgrading from a version pre-0.6.2 then please see the section "Move user data to home directory" in [the 0.6.2 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.6.2.md), and follow the instructions there (except the `commitmentlist` file - that can actually be left alone, the previous release notes were wrong on this point).)
+
+(If you are upgrading from a version pre-0.7.0 please read the "Upgrading" section in [the 0.7.0 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.7.0.md).)
+
+First run the `install.sh` script as mentioned in the README. When prompted to overwrite the directory `jmvenv`, accept.
+
+Note that `./install.sh -?` will show the options for installation.
+
+Notable changes
+===============
+
+### Bugfix - avoid cases of unnecessary transaction signing failure
+
+See [PR 910](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/910). In cases where makers sent data (in the `!ioauth` command, that transfers the information needed to construct the coinjoin) which was not valid in some way, such as the utxo not being valid (e.g. already spent), then sometimes (depending on exact conditions), those maker utxos were *left in* the transaction template that the Taker constructs. This led to a condition where a transaction would simply timeout as unsigned. While this doesn't stop the tumbler from continuing (it tries again after a timeout), it is a bad outcome, not least because PoDLE commitments get used up unnecessarily; the transaction could have completed normally, using the makers with correctly verified data.
+
+After this commit, the code has been refactored to more cleanly organize the process of (a) verify that makers' data is correct and only then (b) add the makers' utxos to the transaction template.
+
+Coinjoins should fail to complete less often after this fix.
+
+`9b01d9a` minor refactor of Taker.receive_utxos()
+
+### Bugfixes to Payjoin code
+
+Three fixes/changes here. The first dealt with a bug where, if the index of the change output were zero, the parameter `additionalfeeoutputindex` was being incorrectly ignored, preventing the possibility of subtracting the fee from the change. This happened about 50% of the time where this option was chosen and led to lower transaction fees than intended.
```

Comment:
time ~where~ when 

_It might be better to use ‘time’ with the time-relative pronoun ‘when’. (Alternatively, use ‘in/on which’.)_

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/912/commits/f1e440cd596e5d46d9da9d6bea0db9e1489d988d